web/.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["es2015", "react"]
+  "presets": ["es2015", "react"],
+  "plugins": ["transform-class-properties"]
```

Comment:
I'm ok with this particular addition (as long as we restrict ourselves to static properties), but we should define a general strategy how we want to handle ES7+ additions. 
I think we enable ES7 in the very few cases that meet the following criteria:

- There is an obvious benefit in readability.
- We can replace the code easily with a corresponding ES6 version.
- The resulting code is relatively self-explaining for someone who knows a tiny bit of Python/Java/C++/.... For instance, I'd probably need to look up what `::this.foo()` does when I see it, whereas static properties are easy to grasp.

@cortesi, is this ok for you?